gobblin-kafka/src/test/java/gobblin/kafka/ErrorManager.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.kafka;
+
+import java.util.Random;
+import java.util.regex.Pattern;
+
+import com.typesafe.config.Config;
+
+import gobblin.util.ConfigUtils;
+
+
+/**
+ * Created by sdas on 7/17/16.
```

Comment:
Primarily to simulate errors thru configuration for the FlakyKafkaProducer. Updating the javadoc.